docs/UserGuide.md

```diff
@@ -580,7 +580,8 @@ Format: `find -t [t/KEYWORD] [st/BOOLEAN]`
 Examples:
 * `find -t st/1` Finds all super-tags.
 * `find -t t/computing` Finds tags that partially contain the word `computing`.
-* `find -t t/friends st/0` Finds regular tags that partially contain the word `friends`.
+* `find -t t/friends st/0` Finds regular tags that partially contain the word `friends`, like the image below.
```

Comment:
Consider changing this to '(Example illustrated below)' as that is used in the other tag commands

Reply:
changed.